tests/unit/helpers/emberTest.js

```diff
@@ -0,0 +1,30 @@
+/*eslint-env node, mocha, es6 */
+'use strict';
+// var rewire = require('rewire');
+// var ember = rewire('../../../lib/helpers/ember.js');
+var ember = require('../../../lib/helpers/ember');
+var expect = require('chai').expect;
+
+describe('ember helper', function () {
+  // before(function () {
+  // });
+
+  it('returns a value', function () {
+    expect(ember).to.be.ok();
+  });
+
+  it('returns local by default on linux', function () {
+    if (process.platform !== 'win32') {
```

Comment:
I'm still figuring all this out, but so far it seems that it's not possible to directly spawn a local (meaning inside the `node_modules` folder within the project) executable because npm does not create the necessary `.cmd` files.  I've submitted https://github.com/timoxley/npm-which/issues/4 to see if `npm-which` or `npm-run` can help us here.  There's got to be some way around this, I can't imagine we're the first to try to do something like this.